Review-11-Partial-Differential-Equations.ipynb

```diff
@@ -382,7 +382,7 @@
     "x_{k+1} = D^{-1}(b - (L + U)x_k)\n",
     "$$\n",
     "\n",
-    "The method will fail outright if $D$ is singular eg $D^{-1}$ does not exist."
+    "The method will fail outright if $D$ is singular eg $D^{-1}$ does not exist or if there is a zero on the diagonal."
```

Comment:
The `or` is redundant in the sense that a diagonal matrix which contains a zero is always singular.